frontend/packages/console-shared/locales/en/console-shared.json

```diff
@@ -133,6 +133,15 @@
   "Increase the Pod count": "Increase the Pod count",
   "Decrease the Pod count": "Decrease the Pod count",
   "Close": "Close",
+  "Quick search bar": "Quick search bar",
+  "Add to Project": "Add to Project",
```

Comment:
This translation probably shouldn't be there. Gotta probably find a contextual reason to have this. This is a Topology use-case.